### ISSUE TITLE
1.7.10 issues

### ISSUE THREAD

**Issue by kuciyo** [hammy2899/ArchonCrates#2]
- sounds not work
- virtual crates gives errors
- prizes not given to players

using dev build but same problems on all other versions
```
Could not pass event PlayerInteractEvent to ArchonCrates v18.0 BETA
org.bukkit.event.EventException
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:509) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:494) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at org.bukkit.craftbukkit.v1_7_R4.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:242) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.PlayerInteractManager.interact(PlayerInteractManager.java:382) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.PlayerConnection.a(PlayerConnection.java:662) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:50) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.PacketPlayInBlockPlace.handle(PacketPlayInBlockPlace.java:86) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.NetworkManager.a(NetworkManager.java:186) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.ServerConnection.c(ServerConnection.java:81) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.MinecraftServer.v(MinecraftServer.java:734) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.DedicatedServer.v(DedicatedServer.java:289) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.MinecraftServer.u(MinecraftServer.java:584) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.MinecraftServer.run(MinecraftServer.java:490) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.ThreadServerApplication.run(SourceFile:628) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
Caused by: java.lang.IllegalArgumentException: No enum constant org.bukkit.Sound.BLOCK_CHEST_OPEN
	at java.lang.Enum.valueOf(Enum.java:238) ~[?:1.8.0_101]
	at org.bukkit.Sound.valueOf(Sound.java:11) ~[spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at com.HamiStudios.ArchonCrates.CustomGUI.VirtualKeyGUI.open(VirtualKeyGUI.java:109) ~[?:?]
	at com.HamiStudios.ArchonCrates.Events.BlockClickEvent.onBlockClick(BlockClickEvent.java:66) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor97.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:298) ~[spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	... 15 more
```
```
[ArchonCrates] Task #914 for ArchonCrates v18.0 BETA generated an exception
java.lang.IllegalArgumentException: No enum constant org.bukkit.Sound.BLOCK_NOTE_BASS
	at java.lang.Enum.valueOf(Enum.java:238) ~[?:1.8.0_101]
	at org.bukkit.Sound.valueOf(Sound.java:11) ~[spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at com.HamiStudios.ArchonCrates.Tasks.Crate$10.run(Crate.java:306) ~[?:?]
	at org.bukkit.craftbukkit.v1_7_R4.scheduler.CraftTask.run(CraftTask.java:71) ~[spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at org.bukkit.craftbukkit.v1_7_R4.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:350) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.MinecraftServer.v(MinecraftServer.java:641) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.DedicatedServer.v(DedicatedServer.java:289) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.MinecraftServer.u(MinecraftServer.java:584) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.MinecraftServer.run(MinecraftServer.java:490) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
	at net.minecraft.server.v1_7_R4.ThreadServerApplication.run(SourceFile:628) [spigot.jar:git-Spigot-1.7.9-R0.2-208-ge0f2e95]
```

**Comment by y0hami** [comment]
Hi kuciyo,

I'm currently working on a new release, these bugs should be fixed when its released!

**Comment by y0hami** [comment]
Fixed in new release (19.0.1).
